app/Controllers/authController.php

```diff
@@ -148,6 +148,8 @@ public function formLoginAction() {
 					FreshRSS_FormAuth::deleteCookie();
 				}
 
+				Minz_Translate::init($conf->language);
```

Comment:
Is it to fix a bug, or for the new language detection?

Reply:
It's to fix the translation for the message underneath.

Reply:
Ok :-)